sdk/python/jobs/automl-standalone-jobs/automl-image-instance-segmentation-task-fridge-items/automl-image-instance-segmentation-task-fridge-items.ipynb

```diff
@@ -864,6 +947,92 @@
     "hd_job"
    ]
   },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 4.3.1 Manual hyperparameter sweeping for models from MMDetection (Preview)\n",
+    "\n",
+    "Similar to how you can use any model algorithm from MMDetection for individual runs, you can also include these models to perform a hyperparameter sweep. You can also choose a combination of model algorithms supported natively by [AutoML](https://docs.microsoft.com/en-us/azure/machine-learning/how-to-auto-train-image-models?tabs=CLI-v2#configure-model-algorithms-and-hyperparameters) and model algorithms from [MMDetection](https://github.com/open-mmlab/mmdetection/blob/v2.28.2/docs/en/model_zoo.md).\n",
+    "\n",
+    "In this example, we sweep over `mask_rcnn_swin-t-p4-w7_fpn_1x_coco`, `cascade_mask_rcnn_r50_fpn_1x_coco`, `scnet_r50_fpn_1x_coco`, and `maskrcnn_resnet50_fpn`, models choosing from a range of values for learning_rate, optimizer, etc., to generate a model with the optimal 'MeanAveragePrecision'.."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create the AutoML job with the related factory-function.\n",
+    "\n",
+    "image_instance_segmentation_job = automl.image_instance_segmentation(\n",
+    "    compute=compute_name,\n",
+    "    # name=\"dpv2-image-instance-segmentation-job-02\",\n",
+    "    experiment_name=exp_name,\n",
+    "    training_data=my_training_data_input,\n",
+    "    validation_data=my_validation_data_input,\n",
+    "    target_column_name=\"label\",\n",
+    "    primary_metric=InstanceSegmentationPrimaryMetrics.MEAN_AVERAGE_PRECISION,\n",
+    "    tags={\"my_custom_tag\": \"My custom value\"},\n",
+    ")\n",
+    "\n",
+    "image_instance_segmentation_job.set_limits(\n",
+    "    timeout_minutes=60,\n",
+    "    max_trials=10,\n",
+    "    max_concurrent_trials=2,\n",
+    ")\n",
+    "\n",
+    "image_instance_segmentation_job.extend_search_space(\n",
+    "    [\n",
+    "        SearchSpace(\n",
+    "            model_name=Choice([\"maskrcnn_resnet50_fpn\"]),\n",
+    "            learning_rate=Uniform(0.0001, 0.001),\n",
+    "            # warmup_cosine_lr_warmup_epochs=Choice([0, 3]),\n",
```

Comment:
remove/uncomment this line?

Reply:
They were originally added to specify that, you can also use these hyperparameters with the models. But the original cell that does manual hyperparameter sweep has these comments. So, it makes sense to remove from here. Will do.